src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/DateTimeValueResolver.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;
 
+use Symfony\Component\Clock\ClockInterface;
```

Comment:
After review, I think we can now typehint the PSR interface.
```suggestion
use Psr\Clock\ClockInterface;
```

Reply:
indeed!